src/user/commands/get-user-info.command.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/database/prisma.service';
+import { UserInfoDTO } from '../dto/user-info.dto';
+
+@Injectable()
+export class GetUserInfo {
+  constructor(private prisma: PrismaService) {}
+
+  async execute(UserId: number): Promise<UserInfoDTO> {
```

Comment:
Nome de variável ou parâmetro sempre começa com letra minúscula:

```suggestion
  async execute(userId: number): Promise<UserInfoDTO> {
```